src/components/Select/types.ts

```diff
@@ -222,14 +222,25 @@ export type TSelectWrapperProps = {
   type?: 'primary' | 'secondary' | 'tertiary'
 }
 
-declare type TNavItemValue = {
+export type TProfileDropdownBodyItems = {
   icon?: TSVGIconComponent
-  text: string
+  text?: string
```

Comment:
I think you can delete both text and children from TProfileDropdownBodyItems type.

Reply:
❤️